tools/probes/probes_main.c

```diff
@@ -111,10 +121,13 @@ int init_wave(struct wave_files *files, uint32_t buffer_id, uint32_t format)
 		exit(0);
 	}
 
-	fprintf(stdout, "%s:\t Creating wave file for buffer id: %d\n",
-		APP_NAME, buffer_id);
+	fprintf(stdout, "%s:\t Creating %s file for buffer id: %d\n",
+		APP_NAME, audio ? "wave" : "raw", buffer_id);
 
-	sprintf(path, "buffer_%d.wav", buffer_id);
+	if (audio)
+		sprintf(path, "buffer_%d.wav", buffer_id);
+	else
+		sprintf(path, "buffer_%d.raw", buffer_id);
```

Comment:
nitpick: `sprintf(path, "buffer_%d.%s", buffer_id, audio ? "wave" : "raw");`

but your version is likely more readable..

Reply:
Fixed.